torch/onnx/diagnostic/rules.yaml

```diff
@@ -0,0 +1,81 @@
+# Rules are based on sarif ReportingDescriptor format.
+# TODO: Define additional format requirements on top of sarif for our usage.
+#       For example: pre defined keys for message_strings for logging on different levels.
+# TODO: Based on above, create helper script to generate new rules.
+# TODO: Separate rules into individual files.
+
+- id: RL0001
```

Comment:
What is RL?

Reply:
Some random placeholder, updated to POE(PyTorch ONNX Exporter).